### PR TITLE
Add desktop-Version

### PR DESCRIPTION
**Pull request by yohannesdagnachew**
- Add HTML to the desktop version
- Add CSS to the desktop version

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,438 +1,974 @@
...
+    left: 0;
+    top: 0;
+    font-family: "Crete Round", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 2.5rem;
+    line-height: 3.25rem;
+    align-items: center;
+    color: #091e42;
+  }
```
Comment (KayLemba) [inline style.css:597]: Please kindly fix the margin and padding on the recent works title and please lets stick to the design on Figma.

This is your design:
Screen Shot 2022-04-29 at 4 50 59 PM

Figma design:


Screen Shot 2022-04-29 at 4 51 30 PM

Your recent works title should be exactly as it is in the figm template please let work on that and carefully inspect the template

**Review by KayLemba — CHANGES_REQUESTED** [review]
Hi @yohannesdagnachew    👋
Good job so far working on the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉
No linter errors. ✔️
GitHub flow is followed. ✔️

Required Changes ♻️
Check the comments under the review in the inline comments.
 - Please add Proffessional and descriptive PULL REQUEST 
- Please add a screenshot showcasing your UI and a project description giving details about the project you are building  to your README file to make it look more proffessional lets take making a proffessional README FILE seriously in all projects we build please have a look at this link it will help you build a good and proffessional README  [
](https://github.com/microverseinc/readme-template) lets not overlook this for it is a very important requirement thanks.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css (hunk 1 above)
Comment (yohannesdagnachew) [reply style.css:597]: Hi @KayLemba 
Thanks for the review.
But I don't understand some of your comments about "Please add a screenshot showcasing your UI and a project description giving".

Comment (KayLemba) [reply style.css:597]: Add a screen shot of your current webpage which is the desktop version add it to your Readme file look at the link I shared with you on the readme template

Comment (KayLemba) [reply style.css:602]: Then a project description is details about the project for example you can say: "This is a desktop version portfolio.... etc write about the project you are building look at the link about the Readme file I shared with u

Comment (yohannesdagnachew) [reply style.css:602]: Hi @KayLemba 
I think it is not on the requirement please check it out, Sir.

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @yohannesdagnachew  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Fixed issues addressed by previous reviewer 👍 
- Linter checks passing ✔️ 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**